apps/src/lib/levelbuilder/lesson-editor/StandardsEditor.jsx

```diff
@@ -135,10 +145,70 @@ class StandardsEditor extends Component {
     this.handleRemoveStandardDialogClose();
   };
 
+  handleSelectFramework = e => {
+    const frameworkShortcode = e.target.value;
+    this.setState({frameworkShortcode});
+  };
+
+  constructStandardOption = standard => ({
+    value: standard.shortcode,
+    label: `${standard.frameworkShortcode.toUpperCase()} - ${
+      standard.shortcode
+    } - ${standard.description}`,
+    standard: standard
+  });
+
+  constructSearchOptions = json => {
+    const existingShortcodes = this.props.standards.map(
+      standard => standard.shortcode
+    );
+    const standards = json
+      // Filter any that are already added to lesson
+      .filter(standard => !existingShortcodes.includes(standard.shortcode))
+      .map(standard => this.constructStandardOption(standard));
+    return {options: standards};
+  };
+
+  onSearchSelect = option => {
+    this.props.addStandard(option.standard);
+  };
+
   render() {
     const columns = this.getColumns();
     return (
       <div>
+        <label>
+          <strong>Filter by framework</strong>
+        </label>
+        <select onChange={this.handleSelectFramework} style={styles.select}>
+          <option value="">(none)</option>
+          <option value="iste">ISTE Standards for Students</option>
```

Comment:
Should this list be pulled from somewhere?

Reply:
oops, yes: https://codedotorg.atlassian.net/browse/PLAT-885